nginx/lib.sls

```diff
@@ -20,3 +20,13 @@
 
     {{ ret|json() }}
 {%- endmacro -%}
+
+{%- macro configurable(options, key, default_value) -%}
```

Comment:
This deals with a Jinja limitation I ran into regarding falling back to defaults.

```
dict.get('key', 'default')
```

causes an exception if `dict` is unset. Which could be, since someone could enable this feature but not configure it; therefor fall back to the defaults.

```
dict.key|default('default')
```

didn't work either because of the same reason.

I even tried

```
dict.get('key', default')|default('default')
```

which would probably work, but results in repeating myself in regards to default values and doesn't work either if the `dict` is undefined.